src/SignalRServiceExtension/TriggerBindings/SignalRTriggerBindingProvider.cs

```diff
@@ -125,15 +132,32 @@ private void ValidateSignalRTriggerAttributeBinding(SignalRTriggerAttribute attr
 
         private string GetCategoryFromMethodName(string name)
         {
-            if (string.Equals(name, Event.Connected, StringComparison.OrdinalIgnoreCase) ||
-                string.Equals(name, Event.Disconnected, StringComparison.OrdinalIgnoreCase))
+            if (string.Equals(name, Constants.OnConnected, StringComparison.OrdinalIgnoreCase) ||
+                string.Equals(name, Constants.OnDisconnected, StringComparison.OrdinalIgnoreCase))
```

Comment:
hard code map to function name might be easily break and hard for customer to identify problem. Any way to notice the method name mapping logic?